tests/Server/TextDocument/CompletionTest.php

```diff
@@ -214,6 +214,23 @@ public function testUsedClass()
         ], true), $items);
     }
 
+    public function testGlobalConstant() {
+        $completionUri = pathToUri(__DIR__ . '/../../../fixtures/completion/constant_with_namespace.php');
+        $this->loader->open($completionUri, file_get_contents($completionUri));
+        $items = $this->textDocument->completion(
+            new TextDocumentIdentifier($completionUri),
+            new Position(11, 7)
+        )->wait();
+
+        var_dump($items);
+        $this->assertEquals(new CompletionList([
+            new CompletionItem(
+                'HELLO',
+                CompletionItemKind::VARIABLE
+            )
```

Comment:
I'm doing some research why this doesn't want to suggest it, but when using it without phpunit, it suggests beautiful.